geography/README.md

```diff
@@ -0,0 +1,148 @@
+| `description`      | String    | Optional   | Detailed description of geography                                                                      |
+| `geography_id`     | UUID      | Required   | Unique ID of geography                                                                 |
+| `geography_json`   | UUID      | Required   | The GeoJSON that defines the geographical coordinates.
+| `effective_date`   | [timestamp][ts] | Optional   | The date at which a Geography is considered "live".  Must be at or after `publish_date`.
+| `publish_date`     | [timestamp][ts] | Required   | Timestamp that the policy was published, i.e. made immutable                                             |
+| `prev_geographies` | UUID[]    | Optional   | Unique IDs of prior geographies replaced by this one                                   |
+
+<a name="file-format"></a>
+
+## File format
+
+To use flat files rather than REST endpoints, Geography objects should be stored in `geographies.json`.  The `geographies.json` file will look like the output of `GET /geographies`.  
+
+Example `geographies.json`
+```json
+{
+    "version": "0.4.0",
+    "updated:" "1570035222868",
+    "geographies": [
+        {
+            // GeoJSON 1
+        },
+        {
+            // GeoJSON 2
+        }
+    ]
+}
+```
+
+<a name="endpoints"></a>
+
+## REST Endpoints
+
+Responses must set the `Content-Type` header, as specified in the [Provider versioning](../provider/README.md#versioning) section. They must also specify the API version in the JSON-formatted response body, under the `version` key.
+
+The Geography Author API consists of the following endpoints:
+
+**Endpoint**:  `/geographies/{geography_id}`
+
+**Method**: `GET`
+
+Path Params:
+
+| Name          | Type | Required/Optional | Description                                       |
+| ------------- | ---- | --- | --------------------------------------------------- |
+| geography_id  | UUID | Required   | Unique identifier for a single specific Geography |
+
+Returns: A single Geography.  
+
+Response body:
+```js
+{
+  version: '1.0.0',
+  geography: {
+    geography_id: UUID,
+    geography_json: GeoJSON FeatureCollection,
+    prev_geographies: UUID[],
+    name: string,
+    publish_date: timestamp
+    effective_date: timestamp
+    description: string
+  } 
+}
+```
+
+Response codes:
+- 200 - success
+- 401 - unauthorized
+- 404 - no geography found
+- 403 - user is attempting to read an unpublished geography, but only has the `geographies:read:published` scope.
+
+**Endpoint**:  `/geographies`
+**Method**: `GET`
+
+Path Params: 
+
+| Name         | Type      | Required/Optional | Description                                    |
+| ------------ | --------- | --- | ---------------------------------------------- |
+| `summary`    | string    | Optional   | Return geographies, minus the GeoJSON in each geography object     |
```

Comment:
Bit of a nit, but I feel it could be helpful to show an example of a summarized `Geography`.

Reply:
I'd love to see some examples, maybe in an examples folder like we have with [Policy](https://github.com/openmobilityfoundation/mobility-data-specification/tree/main/policy/examples), or maybe just an examples.md file. Do you want to create that?

Reply:
I created the start of an examples.md file for others to add on to. 

https://github.com/openmobilityfoundation/mobility-data-specification/blob/feature-geography/geography/examples.md

Reply:
Will move this into a folder like in Policy and will add some real geographic data for some concrete examples and visuals of the GeoJSON.